### ISSUE TITLE
Warning during compiling from source

### ISSUE THREAD

**Issue by proguard** [vi/websocat#61]
Hi,

We have found this to be a great program. while compling to ARM (aarch64), we get this warning

Compiling smart-default v0.3.0
   Compiling derivative v1.0.2
   Compiling structopt v0.2.16
   Compiling websocat v1.5.0 
warning: use of deprecated item 'try': use the `?` operator instead
   --> src/net_peer.rs:179:9
    |
179 |         try!(self.0.shutdown(std::net::Shutdown::Write));
    |         ^^^
    |
    = note: `#[warn(deprecated)]` on by default

warning: use of deprecated item 'try': use the `?` operator instead
   --> src/unix_peer.rs:317:9
    |
317 |         try!(self.0.shutdown(std::net::Shutdown::Write));
    |         ^^^

    Finished release [optimized] target(s) in 8m 45s

Would this cause errors for Websocat durining operation?

Thanks
Aguard

**Comment by vi** [comment]
No, this is a minor warning. It is already fixed in pending websocat 2.0

**Comment by proguard** [comment]
Thank you.

Great work.
